src/darwin/Framework/CHIP/CHIPThreadOperationalDataset.h

```diff
@@ -0,0 +1,76 @@
+/**
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface CHIPThreadOperationalDataset : NSObject
+
+extern size_t const CHIPSizeThreadNetworkName;
+extern size_t const CHIPSizeThreadExtendedPanId;
+extern size_t const CHIPSizeThreadMasterKey;
+extern size_t const CHIPSizeThreadMeshLocalPrefix;
+extern size_t const CHIPSizeThreadPSKc;
```

Comment:
These should be documented.  In particuluar, the fact that they have something to do with `initWithNetwork` and the sizes of its args?